app/views/static_pages/home.html.erb

```diff
@@ -1,4 +1,3 @@
-<% provide(:title, t("static_pages.home.title")) %>
 <h1><%= t("static_pages.home.title") %></h1>
 <p>
   The <a href=<%= t("static_pages.link.home") %>><em>Ruby on Rails
```

Comment:
1. Line 1: remove dấu ()
2. Sao lại fix I18n lưng chừng đồi ri em.
`The `, `Ruby on Rails`

Tốt nhất nên gộp nguyên câu lại thành 1 mesage thôi em nhé. Tìm hiểu biến trong I18n và fix chỗ này nhé